Classes/Importer/AbstractImporter.php

```diff
@@ -383,6 +425,15 @@ protected function unsetAllNodeTemplateProperties(NodeTemplate $nodeTemplate)
     }
 
     /**
+     * @param array $data
+     * @return array
+     */
+    protected function getPropertiesFromDataProviderPayload(array $data)
```

Comment:
Sometimes the properties of the node can be store at the second level of the array, the first can be reserved for metadata, so with this method you can override the default behaviour to get node properties at the first level of the array